### PR TITLE
Feature/extra questions summary

### PR DESCRIPTION
**Pull request by boris-arkenaar**
Show the answers, to additional questions that came from the backend, in the final preview step of the incident wizard.

### CODE REVIEW

File: src/signals/incident/definitions/wizard-step-2-vulaan/__mocks__/afval.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { Validators } from 'react-reactive-form';
```
Comment (aditudorache) [inline src/signals/incident/definitions/wizard-step-2-vulaan/__mocks__/afval.js:1]: Where is this mock used?

File: src/signals/incident/components/IncidentWizard/index.js [hunk 2]
```diff
@@ -36,20 +36,24 @@ const IncidentWizard = ({ wizardDefinition, getClassification, updateIncident, c
...
                     id={`incident/${key}`}
                     render={() => {
-                      const { form, formFactory, label, postponeSubmitWhenLoading, preview } = wizardDefinition[key];
-                      const shouldRender = preview || form || formFactory;
+                      const {
+                        form,
+                        formFactory,
+                        label,
+                        postponeSubmitWhenLoading,
+                        preview,
```
Comment (aditudorache) [inline src/signals/incident/components/IncidentWizard/index.js:44]: Is this `preview` still used? I would remove it.

File: src/signals/incident/definitions/wizard-step-5-samenvatting.js [hunk 3]
```diff
@@ -57,6 +60,56 @@ export const summary = controls =>
     {}
   );
 
+const expandQuestions = memoize(
```
Comment (aditudorache) [inline src/signals/incident/definitions/wizard-step-5-samenvatting.js:63]: Why do we need a memoize here?

Comment (boris-arkenaar) [reply src/signals/incident/definitions/wizard-step-5-samenvatting.js:63]: So it won't break the `react-reactive-form`.

File: src/signals/incident/definitions/wizard-step-2-vulaan/__mocks__/afval.js (hunk 1 above)
Comment (boris-arkenaar) [reply src/signals/incident/definitions/wizard-step-2-vulaan/__mocks__/afval.js:1]: [Here](https://github.com/Amsterdam/signals-frontend/blob/01c26302656341a97bc0bf945205d21e35500f32/src/signals/incident/definitions/__tests__/wizard-step-5-samenvatting.test.js#L19)

Comment (aditudorache) [reply src/signals/incident/definitions/wizard-step-2-vulaan/__mocks__/afval.js:1]: Not sure I understand, the `extra_bedrijven_horeca_wat` is mocked in the afval component but comes from `overlast-bedrijven-en-horeca`. Something is not matching there. Can you take a look again? 
A mock shouldn't necessary if you check on the actual values from `afval` category
```
      const expected = expect.objectContaining({
        vulaan: {
          extra_afval: {
            label: 'Waar vandaan',
            optional: true,
            render: Label,
          },
          extra_container_kind: {
            label: 'Soort container',
            optional: true,
            render: Label,
          },
          extra_container_number: {
            label: 'Container nummer',
            optional: true,
            render: Label,
          },
        },
```

Comment (boris-arkenaar) [reply src/signals/incident/definitions/wizard-step-2-vulaan/__mocks__/afval.js]: You're right, the mock isn't necessary! 👍

**Review by aditudorache — APPROVED** [review]

**Review by MauritsR — APPROVED** [review]

**Pull request merged**